### ISSUE TITLE
Would be much nicer if we could pass in our own logger

### ISSUE THREAD

**Issue by nickminutello** [carlos8f/haredis#19]
The haredis logging is good - but the problem I face is that its disconnected from the rest of our logging. I use log4js ... but should be able to pass in any logger that implements debug, info, warn, error, etc

**Comment by carlos8f** [comment]
good idea, i wanted to do this but never got around to it. i'll accept a pull request or implement it as time permits.

**Comment by nickminutello** [comment]
Cool. I will have a stab if I get some free time
